core/.env_sample

```diff
@@ -1,8 +1,13 @@
-MNEMONIC=""           # truffle wallet mnemonic. Used in all scripts, tools & cli.
-SLACK_WEBHOOK=""      # OPTIONAL: webhook to send messages to a slack channel. See Logger.js
+MNEMONIC=""             # truffle wallet mnemonic. Used in all scripts, tools & cli.
+# To enable slack messaging from winston a slack webhook is needed.
+SLACK_WEBHOOK=""        # Webhook to send messages to a slack channel. See Logger.js
```

Comment:
Why is this no longer optional?

Reply:
They are all optional. I removed that comment to just make things a bit more consistent.